From 58b89674b46649a6c6a70387510feac0292c8164 Mon Sep 17 00:00:00 2001
From: Tom St Denis <tom.stdenis@amd.com>
Date: Tue, 6 Sep 2016 12:07:21 -0400
Subject: [PATCH 0879/1722] drm/amd/amdgpu: Comment out currently unused SI DPM
 struct

The tahiti_le structure is not currently used.  Comment it out
to avoid warnings.

Signed-off-by: Tom St Denis <tom.stdenis@amd.com>
Acked-by: Edward O'Callaghan <funfunctor@folklore1984.net>
Reviewed-by: Huang Rui <ray.huang@amd.com>
Reviewed-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Kalyan Alle <kalyan.alle@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/si_dpm.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/amd/amdgpu/si_dpm.c b/drivers/gpu/drm/amd/amdgpu/si_dpm.c
index 7b9debe..92a0417 100644
--- a/drivers/gpu/drm/amd/amdgpu/si_dpm.c
+++ b/drivers/gpu/drm/amd/amdgpu/si_dpm.c
@@ -340,6 +340,7 @@ static const struct si_dte_data dte_data_tahiti =
 	false
 };
 
+#if 0
 static const struct si_dte_data dte_data_tahiti_le =
 {
 	{ 0x1E8480, 0x7A1200, 0x2160EC0, 0x3938700, 0 },
@@ -357,6 +358,7 @@ static const struct si_dte_data dte_data_tahiti_le =
 	85,
 	true
 };
+#endif
 
 static const struct si_dte_data dte_data_tahiti_pro =
 {
-- 
2.7.4

